lib/synapse/haproxy.rb

```diff
@@ -386,6 +388,7 @@ class Haproxy
         "option accept-invalid-http-request",
         "option accept-invalid-http-response",
         "option allbackups",
+        "option allredisp",
```

Comment:
should we try to re-sync these lists of options with latest haproxy? i wonder what other options have been added since i put these lists in here...